test/jobs.test.js

```diff
@@ -41,23 +50,45 @@ describe('Jobs Unit Test', () => {
         mockery.disable();
     });
 
+    describe('redis constructor', () => {
+        it('creates a redis connection given a valid config', () => {
+            assert.calledWith(mockRedis, null);
```

Comment:
It constructs it with the `connectionDetails`. This shouldn't be called with `null`